plenoptic/synthesize/eigendistortion.py

```diff
@@ -203,82 +186,75 @@ def synthesize(self, method='power', e_vecs=None, tol=1e-10, n_steps=1000, seed=
             Index of each eigendistortion/eigenvalue. This points to the `synthesized_eigenindex` attribute of the
             object.
         """
-        self._set_seed(seed)
-        self.store_progress = store_progress
 
-        if e_vecs is None:
-            e_vecs = []
+        seed = np.random.randint(0, 2**32-1) if seed is None else seed
+        torch.manual_seed(seed)
```

Comment:
Three points here:
1. Nothing in this depends on numpy RNG does it? [PyTorch docs](https://pytorch.org/docs/stable/notes/randomness.html#random-number-generators-in-other-libraries) discuss setting numpy RNG as well. It says "However, some applications and libraries may use NumPy Random Generator objects, not the global RNG (https://numpy.org/doc/stable/reference/random/generator.html), and those will need to be seeded consistently as well.", but I think as long as the only randomness-related code we use is part of pytorch, we're good.

2. There is a `Synthesis._set_seed(seed)` function that sets both pytorch and numpy's RNG. Though it's `None` behavior is different.

3. What's the point of setting the seed using a random int (which itself will be based on the state of the RNG), rather than just not setting it? Does seed get stored in some way such that it's accessible to the user after the fact? (so they wouldn't have to set it when initializing, but could reproduce it afterwards)?

Reply:
1. Nothing here dpeends on numpy RNG. 
2. Prev. implementation used ``Synthesis._set_seed()`` but was replaced with this in the rollback from subclassing Synthesis.
3. No real point. Changed to just not setting it now.